cleanlab/datalab/internal/issue_manager/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Union, Optional, Any
+from scipy.sparse import csr_matrix
+
+
+class ConstructedKNNGraph:
+    """construct KNN Graph"""
+
+    def __init__(self, datalab):
+        self.datalab = datalab
+
+    def process_knn_graph_from_inputs(self, kwargs: Dict[str, Any]) -> Union[csr_matrix, None]:
```

Comment:
We can replace `Union[csr_matrix, None]` with `Optional[csr_matrix]` for consistency. [Reference](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type). 
```suggestion
    def process_knn_graph_from_inputs(self, kwargs: Dict[str, Any]) -> Optional[csr_matrix]:
```